collector/zfs_linux.go

```diff
@@ -235,3 +261,35 @@ func (c *zfsCollector) parsePoolObjsetFile(reader io.Reader, zpoolPath string, h
 
 	return scanner.Err()
 }
+
+func (c *zfsCollector) parsePoolStateFile(reader io.Reader, zpoolPath string, handler func(string, string, uint64)) error {
```

Comment:
What is the reason for passing in the handler func instead of calling it directly?

Reply:
I just followed style in collector/zfs_linux.go.

Reply:
@discordianfish Could you please what I should do?

Reply:
@discordianfish Hello, could you please describe a little bit what I should do?

Reply:
Actually that's alright I think.